apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -19,6 +9,16 @@ document.addEventListener('DOMContentLoaded', function () {
   const hamburgerButtons = document.querySelector('#hamburger-sign-up-buttons');
   const signUpPage = window.location.pathname.includes('/users/sign_up');
 
+  const isInCreateAccountButtonExperiment = statsigReporter.getIsInExperiment(
+    'create_account_button_2',
+    'showCreateAccountButton',
+    false
+  );
+
+  function showCreateAccountButton() {
```

Comment:
is this function still necessary? (though if you don't need to make another update, it's probably not worth updating this)

Reply:
Probably not, but to your point, drone is almost done so I'll leave it 😅